src/routes/reviewRouter.js

```diff
@@ -0,0 +1,11 @@
+const router = require("express").Router();
+
+const { reviewController } = require("../controllers");
+const { upload } = require("../middlewares");
+const { loginRequired } = require("../middlewares");
+
+router
+  .route("/new/:storeId")
+  .post(upload.array("reviewImg", 10), loginRequired, reviewController.postNewReview);
```

Comment:
`POST :8000/stores/1/review`

```suggestion
router.post("/:storeId", upload.array("reviewImg", 10), loginRequired, reviewController.postNewReview);
```